entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java

```diff
@@ -176,15 +182,49 @@ public void storeMultipleEntities(final List<SolrEntity<? extends Entity>> solrE
           String.format("Error while searching Solr for entityId=%s", entityId), ex);
     }
 
-    DocumentObjectBinder binder = new DocumentObjectBinder();
     SolrDocumentList docList = rsp.getResults();
-
     if (docList == null || docList.size() == 0) return null;
 
     SolrDocument doc = docList.get(0);
-    return binder.getBean(classType, doc);
+    return toSolrEntity(new DocumentObjectBinder(), doc);
+  }
+
+  
+//  @SuppressWarnings({ "rawtypes", "unchecked" })
+  public List<SolrEntity<? extends Entity>> search(SolrQuery query) throws SolrServiceException {
```

Comment:
This method loads all search results in memory. Not an issue for tests, however it will be problematic for production.

Consider using `SolrSearchCursorIterator` instead. 

Also consider removing this implementation, so it's not accidentally used in live code. _Even if that's unlikely because of our rigorous reviews_ 😌